### DISCUSSION TITLE
Select or unSelect the Switch management network (type "Management only")

### DISCUSSION THREAD

**Discussion by lejibxl** [mattieserver/netbox-topology-views#385] [Q&A]
Is it possible to provide (or perhaps it already exists) the possibility of filtering the schematic Management links?
The "Switch management network" is a kind of separate network with its own structure.

**Comment by dreng** [comment]
I'm not sure if I understand correctly. Are you talking about a management VLAN, OOB-Management or something else?

In general, it is only possible to show connections for all ports and interfaces of a displayed device, not just some of them. We always filter whole devices, no VLANs or interfaces.

**Reply by lejibxl** [reply]
I'm sorry, my English isn't very good,
What I mean is that management port connections (MGMT), which are often part of a separate network (OOB-Management), are found on the same diagram and add a lot of confusion. 
As this type of connection is well differentiated in Netbox via the "Management Only" flag, couldn't we provide a filter at this level?
